Content.Shared/Storage/EntitySystems/DumpableSystem.cs

```diff
@@ -128,58 +129,69 @@ private void StartDoAfter(EntityUid storageUid, EntityUid targetUid, EntityUid u
             delay += itemSize.Weight;
         }
 
-        delay *= (float) dumpable.DelayPerItem.TotalSeconds * dumpable.Multiplier;
+        delay *= (float)comp.DelayPerItem.TotalSeconds * comp.Multiplier;
 
         _doAfterSystem.TryStartDoAfter(new DoAfterArgs(EntityManager, userUid, delay, new DumpableDoAfterEvent(), storageUid, target: targetUid, used: storageUid)
         {
             BreakOnMove = true,
             NeedHand = true,
         });
     }
-
-    private void OnDoAfter(EntityUid uid, DumpableComponent component, DumpableDoAfterEvent args)
+    private void OnDoAfter(EntityUid uid, DumpableComponent comp, DumpableDoAfterEvent args)
     {
-        if (args.Handled || args.Cancelled || !TryComp<StorageComponent>(uid, out var storage) || storage.Container.ContainedEntities.Count == 0)
+        if (args.Handled
+        || args.Cancelled
+        || args.Args.Target == null
+        || !TryComp<StorageComponent>(uid, out var storage)
+        || storage.Container.ContainedEntities.Count == 0)
```

Comment:
Usually multi line expressions in the project are broken up with the operator at the end of the line, not the start.
```suggestion
        if (args.Handled ||
            args.Cancelled ||
            args.Args.Target == null ||
            !TryComp<StorageComponent>(uid, out var storage) ||
            storage.Container.ContainedEntities.Count == 0)
```